cfpb/consumerfinance.gov
# Fix event image/venue map logic

This PR removes several fields and moves another in order to simplify and clarify the logic around selecting a venue image and/or generated map for an Event page. It also fixes an issue where a large CFPB logo image was shown for all events once concluded, regardless of selections made in any of these fields.

---



## Removals

- Unused but visible `archive_image` field
- Unnecessary `post_event_image` field and related unit test

## Changes

- Moved `venue_image` tab to the General Content tab and relabeled


## How to test this PR

1. localhost
2. Create or edit an Event page
3. Use the moved "Image" fields in the General Content tab. Note that maps don't appear to render on localhost, but you will see a change in the spacing of the image area that indicates an attempt to generate


## Notes and todos

- This is not intended to be a perfect or complete solution to Event page woes but instead an iteration that fixes a longstanding problem and removes some cruft. There are many other changes we _could_ integrate into a PR like this, but I think we should wait for any larger-scale changes to code until we have a more focused effort to review it.


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
